tests/test_vector2_ctor.py

```diff
@@ -38,3 +40,9 @@ def test_ctor_noncopy_subclass():
 def test_ctor_noncopy_superclass():
     v = Vector2(1, 2)
     assert V(v) is not v
+
+
+@given(v=vectors())
+def test_ctor_pickle(v: Vector2):
```

Comment:
This should likely take a `cls` argument and be parametrised over `Vector2` and a dummy subclass of it, to check we correctly (un)pickle derived classes.